tests/integration/ha_tests/test_scaling_three_units.py

```diff
@@ -60,9 +61,8 @@ async def test_build_and_deploy(ops_test: OpsTest, charm) -> None:
     [
         ["primaries"],
         ["sync_standbys"],
-        ["replicas"],
-        ["primaries", "replicas"],
-        ["sync_standbys", "replicas"],
+        ["primaries", "sync_standbys"],
+        ["sync_standbys", "sync_standbys"],
```

Comment:
No more replicas by default.

Reply:
We should still test non-default case, IMHO.
It will exist in the wild (due to the performance requirements).

Reply:
I also think this is important.

Reply:
Adding a separate test for async.